CIP-0006/README.md

```diff
@@ -29,7 +34,10 @@ Many additional attributes can be envisioned for future wallets, pool explorers,
 
 ## Specification
 
+> **Note** Updated: 2020-11-24 2nd key-pair for validation 2021-02-08 json schema
+
 ### On Chain referenced (main) metadata file
```

Comment:
Are we going to enforce the any CIP which has on-chain data must provide a CDDL file (from #563) - retroactively?
Probably not practical.

Reply:
I think it would make sense to demand CDDL here if there had been a CIP for the original on-chain pool metadata specification, which there wasn't: as far as I know that was only ever documented in apocrypha like SPO setup procedures.

So providing CDDL for the extended structure, but not the original structure, doesn't generally solve that problem.

The alternative would be to broaden the scope of this CIP so it includes _both_ the original metadata spec _and_ the extended spec.  That would avoid having to write an "informational" CIP over 4 years after this method was put into practice: looking a bit silly perhaps.  I would still be in favour of it though: but only in a separate PR because it's beyond the scope of the CIP1 remediation effort.